Alloy/alloy.js

```diff
@@ -130,12 +133,12 @@ Promise
 ///////////////////////////////
 function banner() {
 	var str =
-	'       .__  .__                \n' +
-	'_____  |  | |  |   ____ ___.__.\n' +
-	'\\__  \\ |  | |  |  /  _ <   |  |\n' +
-	' / __ \\|  |_|  |_(  <_> )___  |\n' +
-	'(____  /____/____/\\____// ____|\n' +
-	'     \\/                 \\/';
+		'       .__  .__                \n' +
+		'_____  |  | |  |   ____ ___.__.\n' +
+		'\\__  \\ |  | |  |  /  _ <   |  |\n' +
+		' / __ \\|  |_|  |_(  <_> )___  |\n' +
+		'(____  /____/____/\\____// ____|\n' +
+		'     \\/                 \\/';
```

Comment:
I for one wouldn't miss this ascii art should it go away.

Reply:
I'm also good with that

Reply:
Leave the banner alone. It's outside the context of this PR. We'll fix it later.